.travis.yml

```diff
@@ -1,35 +1,30 @@
 language: php
 
-php:
-# - 5.3 # requires old distro, see below
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
-  - hhvm # ignore errors, see below
-
 # lock distro so new future defaults will not break the build
 dist: trusty
 
-# also test lowest dependencies on PHP 7
 matrix:
   include:
     - php: 5.3
       dist: precise
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
     - php: 7.0
-      env:
-        - DEPENDENCIES=lowest
```

Comment:
Intentionally removed? Not sure how much value this still provides, but the environment variable is still referenced below. Perhaps keep this and discuss this in a separate ticket?

Reply:
Not intentionally removed, added it again